governance/working groups/index.md

```diff
@@ -0,0 +1,91 @@
+
+Therefore it is important to note that establishing a WG with the UK TRE community **does not** imply any outputs of the WG are endorsed by the UK TRE Community.
+
+### Establishing a Working group
+
+1. Member(s) suggesting a working group fill in the [Working Group Charter](working-group-charter.md).
+2. The Charter is reviewed by the [Community Management Working Group]() to ensure it:
+    - Aligns with [community principles]()
+    - Represents new work being undertaken in the community
+3. If the Charter is rejected by the [Community Management Working Group](), feedback is provided to the submitting member(s) on why, and the Working Group returns to step 1.
+4. If the Charter is approved by the [Community Management Working Group](), the Charter is made available for community review, and communicated with the community, via an Issue on the [UK TRE GitHub](https://github.com/uk-tre/community-management) for a period of at least 14 days.
+5. After the review period, the [Steering Group]() will either formally reject or approve the WG creation, in line with the [Consensus, Review and Objection Management Process](https://docs.google.com/document/d/1SxFnmMKcfsYaO4wjHdiBfGOgPATIsTwRKaLbyjoN1pA/edit?usp=sharing).
+6. If the WG is rejected, any outstanding objections requiring review are collated by the [Steering Group]() and shared with the Working Group proposers for updating the charter, and the Working Group returns to step 4.
+7. Once a Working Group is established, they are formally recognised on the [UK TRE Community website](https://www.uktre.org/), and the community is notified of its creation.
+
+### Endorsing Working Group outputs
+
+1. When a WG has outputs ready to share with the community, they notify the [Community Management Working Group]()
+2. The draft outputs are reviewed by the [Community Management Working Group]() to ensure it:
+    - Aligns with [community principles]()
+3. If approved by the [Community Management Working Group](), the draft outputs are made available for community review, and shared with the community, via an Issue on the [UK TRE GitHub](https://github.com/uk-tre/community-management) for a period of at least 28 days.
+4. At the end of this period, based on community input the outputs will be considered by the [Steering Group]() with 3 possible outcomes:
+
+#### Rejection
+
+The [Steering Group]() rejects the draft outputs of the WG.
+
+The [Steering Group]() will collate the reasons for rejection and share these with the WG.
+The WG can decide to either close out the working group, or amend the outputs to resolve any reasons for rejection.
+
+#### Approved for distribution
+
+The [Steering Group]() approves the outputs for distribution. 
+
+The UK TRE community will signpost to the outputs, but will not specifically endorse them.
+The WG can decide to either close out the working group, or amend the outputs to work towards endorsement.
+
+#### Approved and endorsed
```

Comment:
I want to discuss this briefly today before approving. I am unsure where we said for endorsement outputs will need to be presented at a quarterly event and have a follow up review/discussion period